.github/workflows/build-lambda-zip.yml

```diff
@@ -0,0 +1,27 @@
+name: go get build-lambda-zip 
+on:
+  push:
+  schedule:
+    - cron: "7 7 * * *"
```

Comment:
preface: I'm new to GitHub Actions.

Perhaps for my own learning: what's the value in regularly running this action (over on push)?